### ISSUE TITLE
Children´s interstitial lung disease (chILD)

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#6682]
**Preferred term label:**
Children´s interstitial lung disease

**Synonyms**
chILD; chILD syndrome

**Definition (free text, please give PubMed ID)**
Interstital lung disease (ILD) in children; used synonym to Diffuse parenchymal lung disease (DPLD) in children or pediatric DPLD. The terms chILD or chILD syndrome (not preferred) have been coined to memorize easily and to identify a phenotype that requires prompt and expert evaluation.
PMID: 23905526

**Parent term (use hpo.jax.org/app)**
Interstital lung disease

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 26408013

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think that this should be a term in Mondo transfering to https://github.com/monarch-initiative/mondo/issues/4389
